micrometer-commons/src/main/java/io/micrometer/common/annotation/AnnotationHandler.java

```diff
@@ -16,7 +16,7 @@
 package io.micrometer.common.annotation;
 
 import io.micrometer.common.KeyValue;
-import io.micrometer.common.lang.Nullable;
+import org.jspecify.annotations.Nullable;
```

Comment:
Import order should be fixed. I did a find and replace, so it came out like this.